src/maybe_decompress_brotli.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Gets decompressed response from response if the brotli compression is used.
+ * Got package by default supports br encoded contents only for Node.js 11.7.0 or later.
+ * @param {PassThrough} response
+ * @param {boolean} useBrotli
+ * @returns {PassThrough|Stream} - Decompressed response
+ * @ignore
+ */
+module.exports = function maybeDecompressBrotli(response, useBrotli) {
+    const nodeVersion = Number(process.version.match(/^v(\d+\.\d+)/)[1]);
+    const compression = response.headers['content-encoding'];
+
+    if (compression !== 'br' || !useBrotli) {
```

Comment:
When `content-encoding` is `br`, Got just passes the compressed stream without any trouble?

Reply:
Completely rewrote the compression logic.